pkg/interceptors/interceptors.go

```diff
@@ -124,6 +124,9 @@ func Execute(ctx context.Context, client *http.Client, req *triggersv1beta1.Inte
 	if err != nil {
 		return nil, err
 	}
+
+	r.Header.Set("Content-Type", "application/json")
+
```

Comment:
every request may not have content-type as `application/json` there could be chances of having `text/plain` 

so how about getting header first and then if header is empty make default as `"application/json"`

Reply:
Every interceptor request is `application/json`  even for requests coming in `text/plain`.

Reply:
No tests?

Reply:
Added.